CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to Gin-OAuth2
+
+Please open an issue for bugs and feature requests. We are open to Pull Requests, but we would like to discuss features with you.
+If you want to submit a PR, always use feature branches and let people discuss changes in pull requests.
+
+Pull requests should only be merged after all discussions have been concluded and at least 2 reviewers has given their
+**approval**.
+
+## Guidelines
+
+- **every code change** needs a test
```

Comment:
To me it means: no more PRs with no tests, us included. Wouldn't this make sense?